docs: add mention of CLI command to create token on behalf of another user (#15138)
coder/coder: docs/admin/users/sessions-tokens.md

### Generate a long-lived API token on behalf of another user

> <div class="tabs">
> 
> ## CLI
> 
> As of Coder v2.17+, you can create long-lived tokens on behalf of other users if
> your account is `Owner`. Use the following command:
> 
> ```sh
> coder tokens create my-token --user <username>
> ```
> 
> See the full API reference for
> [`coder tokens create`](../../reference/cli/tokens_create.md)
> 
> ## API
> 
> You can also use the REST API to generate a token on behalf of another user. You
> must have the `Owner` role to do this. Use our API reference for more

Reviewer: ```suggestion
You must have the `Owner` role to generate a token for another user.

As of Coder v2.17+, you can use the CLI or API to create long-lived tokens on behalf of other users. Use the API for earlier versions of Coder.

<div class="tabs">

## CLI

```sh
coder tokens create my-token --user <username>
```

See the full CLI reference for
[`coder tokens create`](../../reference/cli/tokens_create.md)

## API

Use our API reference for more
```
